src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -643,6 +658,35 @@ export default {
 				);
 			}
 		},
+		fetchMFILoans() {
+			// Load mfi recommendations loans data
+			if (this.mfiRecommendationsExp) {
```

Comment:
@roger-in-kiva @egoroza Please move this check into the mounted hook so that it wraps the function call for `fetchMFILoans`. Then remove the console log here and the `return Promise.resolve()` at the bottom of this function.